DOCSP-42735: configure cxn (#49)
mongodb/docs-mongoid: source/quick-start-sinatra/configure-mongodb.txt

.. _mongoid-quick-start-sinatra-connect-to-mongodb:

=================================
Configure Your MongoDB Connection
=================================

.. facet::
   :name: genre
   :values: tutorial

.. meta::
   :keywords: test connection, runnable, code example

.. procedure::
   :style: connected

   .. step:: Specify target database in connection string
      
      When connecting to an Atlas cluster, you must specify the database that
      you want to interact with as the default database in your connection string.
      You must add the database name to your connection string **after the hostname**.
      

>       We want to access the ``sample_restaurants`` database for this tutorial.
>       For example, if the connection string for your Atlas cluster is
>       ``"mongodb+srv://user0:pass123@mongo0.example.com/"``, you can
>       specify the target database as shown in the following code:

Reviewer: I think this can be modified to just introduce the code example to avoid using "we" and to make it more closely mirror our other repos.

```suggestion
      The following example specifies the ``sample_restaurants`` database in a connection string:
```

      .. code-block:: none

>          mongodb+srv://user0:pass123@mongo0.example.com/sample_restaurants

Reviewer: ```suggestion
         mongodb+srv://<username>:<password>@mongo0.example.com/sample_restaurants
```

Author: I'll leave the connection string as is an specify that its a sample

   .. step:: Specify connection

      In your project, create the ``config`` directory, then create a
      file in this directory called ``mongoid.yml``.
      

>       Paste the following configuration into the file, making sure to

Reviewer: ```suggestion
      Paste the following configuration into the ``mongoid.yml`` file, making sure to
```
